### DISCUSSION TITLE
Synced interfaces cannot be edited

### DISCUSSION THREAD

**Discussion by JNCone** [bonzo81/netbox-librenms-plugin#76] [Q&A] [answered]
Hello,

first this is a great and very helpfull plugin! When I try to edit a synced interface I get the following error message:
"Unknown field name 'librenms_id' in custom field data."

I use the newest version of netbox: v4.1.10

can this be fixed?

**Comment by bonzo81** [comment, marked as answer]
HI JNCone,

I am trying to replicate this issue in my environment. I think this might happen when the interface(s) is synced when the ```librenms_id``` custom field doesn't exist. Would this match your setup? Are you using the librenms_id custom field?

**Reply by JNCone** [reply]
No I didn't but everything works now after I added this custom field! Thanks a lot!
